chainerio/cache/file_cache.py

```diff
@@ -82,16 +82,30 @@ class FileCache(cache.Cache):
     Stores cache data in local filesystem,
     ``~/.chainer/chainerio/cache`` by default. Cache data is
     automatically deleted after the object is collected. If the
-    process quit (typically by SIGTERM) the cache remains after the
+    process quits (typically by SIGTERM) the cache remains after the
     death of process.
 
+    Arguments:
+        length (int): Length of the cache array. Typically it is
+            the length of Chainer's ``Dataset``.
+        multithread_safe (bool): Defines multithread safety. If this
+            is ``True``, reader-writer locking is introduced behind
+            the cache system. Typically it is for Chainer's
+            ``MultithreadIterator``.
+        do_pickle (bool):
+            Do automatic pickle and unpickle inside the cache.
+        dir (str):
+            The place for cache data in case home directory is not
```

Comment:
The directory where the cache data will be stored. In the default case, the home directory is used. Recommend to fast storage device and avoid using remote filesystems such NFS